src/index.html

```diff
@@ -5,10 +5,98 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"   >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="img_logo" >
+        <img
+         src="./images/logo.png"
+         alt="moyo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_list_item first_link">
+            <a
+            href="#apple"
+            class="nav_link_item is-active"
+            >
+            Apple
+            </a>
```

Comment:
you need to put attributes in a separate rows if it is more then three. in this case you can just:
```html
<a href="#apple" class="nav_link_item is-active">
   Apple
</a>
```
fix it everywhere